spinalcordtoolbox/reports/qc.py

```diff
@@ -61,6 +62,50 @@ class QcImage:
     _seg_colormap = ["#4d0000", "#ff0000"]
     _ctl_colormap = ["#ff000099", '#ffff00']
 
+    def _assign_label_colors_by_groups(self, labels):
+        """
+        This function handles label colors when labels may not be in a single continuous group:
+
+            - Normal vertebral labels:  [2, 3, 4, 5, 6, ...]
+            - Edge case, TotalSegmentator labels: [31, 32, 33, 200, 201, 217, 218, 219]
+
+        We assume that the subgroups of labesl (1: [31, 32, 33, ...], 2: [200, 201], 3: [217, 218, 219, ...])
+        should each be assigned their own distinct colormap, as to
```

Comment:
Oops, I must have gotten distracted mid-sentence.

```suggestion
        should each be assigned their own distinct colormap, as to group them semantically.